src/main/java/org/ngafid/WebServer.java

```diff
@@ -217,6 +217,7 @@ public static void main(String[] args) {
         Spark.post("/protected/monthly_event_counts", new PostMonthlyEventCounts(gson));
         Spark.get("/protected/severities", new GetSeverities(gson));
         Spark.post("/protected/severities", new PostSeverities(gson));
+        Spark.post("/protected/all_severities", new PostAllSeverities(gson));
```

Comment:
This is also defined in your previous PR? Is your dark mode branch use this as its base?